.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,35 @@
+name: Build & Publish FDroid APK
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  Gradle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: setup jdk
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
any particular reason why we user 11 instead of 17 as set in gradle?

Reply:
Nope, my bad. Will amend.

Reply:
Addressed.